### ISSUE TITLE
Get error when booting 

### ISSUE THREAD

**Issue by Thomas-66** [ghostbsd/ghostbsd-src#2]
This  is strange. I have gotten a new battery for the laptop along with new charger. Now battery instructions said to have it get to7% then charge again. Laptop went into hibernation before I could get it charging. Was stuck in hibernation. Finally got that resolved. I think at least because I can now use keyboard. But now I'm unable to boot on up I'm getting this strange error when trying to boot on to GhostBSD. Going to attempt to attach a picture of the issue. Maybe someone can help me figure out what's going on and how to get around this. http://imgbox.com/VVwa0hIn#

**Comment by Thomas-66** [comment]
Last night I had trouble getting laptop out of hibernation. It would boot to a certain degree and quit. Got a silver looking screen. No keys was working or anything. Decided to search web for different things to try. Tried the Fn + F3 and then fn + F4 keys and still nothing. So went into bios and noticed time was off and changed it. Still nothing and changed it back. Even went into bios on the pc to compare the date etc. So all was looking well again with the bios. Tried to change to booting to USB that way could see if it would do anything at all. Went into bios and changed for the USB to boot up first. That didn't work at all either. Just booted up to what I had posted on Telegram and in the link above. Set it back to bood HD first. same thing. 
Saw something similar in Freebsd forum but I hadn't done any installation of any kiind of drives like that other individual had in the Freebsd forum. I had been off the laptop for about three weeks or more. Then I was able to order the battery and charger which I got late yesterday afternoon. Charged the battery and worked fine. Found out that I had over 121 updates to do on it. Got that all done. Then when waiting for that power to get down it went right into hibernation mode and shut down on me. Have been having this ever since last night.

**Comment by wb7odyfred** [comment]
> Last night I had trouble getting laptop out of hibernation. It would boot to a certain degree and quit. Got a silver looking screen. No keys was working or anything. Decided to search web for different things to try. Tried the Fn + F3 and then fn + F4 keys and still nothing. 

I find that coming out of Hibernation, no GUI screen image appears.  I type Ctl-Alt-F2 or Ctrl-Alt-F8 to wake up screen monitor and see the ttyX "login" prompt.  Then I type Ctl-Alt-F9 and the graphic desktop window appears just as it was when I walked out of the room several minutes ago.  GUI Desktop operation is good, after Ctl-Alt-F9.     Fred    ps.  I was playing a youtube video when the computer went to hibernate. (sleep).   When I clicked any key to wake up the process,  I could here the youtube video sound playing,  but just a black screen monitor.  Until I hit Ctl-Alt-F8   Ctl-Alt-F9
Then I could see my desktop and the youtube video playing.   Hope this information helps.  Can give more and test what you suggest.    Fred
